app/scripts/views/base.js

```diff
@@ -762,6 +762,13 @@ define(function (require, exports, module) {
       this._hasNavigated = true;
     },
 
+    navigateViaServer (url) {
```

Comment:
Of those, `navigateAway` and `setDocumentLocation` seem clearest to me. I'll go with `navigateAway` on the basis that `setDocumentLocation` is maybe *too* implementation-detaily.